apps/dialtone-documentation/docs/components/modal.md

```diff
@@ -97,79 +66,331 @@ Although highly versatile, this doesn't mean modal dialogs are fit for all purpo
   <example-modal />
 </code-well-header>
 
-```html
-<aside class="d-modal" id="modal-base" tabindex="-1" role="dialog" aria-labelledby="modal-title" aria-describedby="modal-description" aria-hidden="true">
-  <div class="d-modal__dialog" role="document">
-    <h2 class="d-modal__header" id="modal-title">…</h2>
-    <p class="d-modal__content" id="modal-description">…</p>
+<code-example-tabs
+htmlCode='
+<button type="button" class="base-button__button d-btn d-btn--primary">
+  <span class="d-btn__label base-button__label"> Click to open </span>
+</button>
+<aside id="modal-base" class="d-modal d-m0 d-modal--animate-in" tabindex="-1" role="dialog" aria-labelledby="modal-title" aria-describedby="modal-description" aria-hidden="false">
+  <div class="d-modal__dialog d-modal__dialog--animate-in" role="document">
+    <h2 class="d-modal__header">
+      Example title
+    </h2>
+    <div class="d-modal__content">
+      <p id="modal-description">
+        Sed at orci quis nunc finibus gravida eget vitae est...
+      </p>
+      <p class="d-mt16"><a href="#" class="d-link">Show me a modal banner</a></p>
+    </div>
     <footer class="d-modal__footer">
-      <button class="d-btn" type="button">…</button>
-      <button class="d-btn d-btn--primary" type="button">…</button>
+      <button class="d-btn d-btn--primary" type="button">
+        Save changes
+      </button>
+      <button class="d-btn" type="button">
+        Cancel
+      </button>
     </footer>
-    <button href="#" class="d-modal__close d-btn d-btn--circle d-btn--lg" aria-label="Close"><IconClose /></button>
+    <button class="d-modal__close d-btn d-btn--circle d-btn--lg" aria-label="Close">
+      <span class="d-btn__icon">
+        <span class="d-icon__wrapper">
+          <div aria-busy="true" role="status" aria-label="" class="d-icon d-icon--size-300" style="display: none;">
+            <div
+              class="skeleton-placeholder d-bar-circle skeleton-placeholder--animate"
+              style="animation-delay: 0ms; animation-duration: 1000ms; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%;"
+            ></div>
+          </div>
+          <svg>...</svg>
+        </span>
+      </span>
+    </button>
   </div>
 </aside>
-```
+'
+vueCode='
+<dt-modal
+  title="Example title"
+  close-button-props="Close"
+  :show="isOpen"
+  @update:show="updateShow"
+  copy="Lorem ipsum ..."
+>
+  <template
+    #footer
+  >
+    <dt-button
+      id="cancel-button"
+      :kind="secondaryButtonKind"
+      importance="clear"
+    >
+      Cancel
+    </dt-button>
+    <dt-button
+      id="confirm-button"
+      :kind="$attrs.kind"
```

Comment:
Probably don't want to show $attrs.kind here. Change to a static kind. There are a few other instances of $attrs.kind in other examples so make sure to do it for all.

Reply:
huh typo, tnk